solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/create.test.ts

```diff
@@ -0,0 +1,166 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import ClientSession from '../../../support/clientSession';
+import Setup from '../../../support/setup';
+import HttpError from '../../../support/utils/HttpError';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('create environment type configs', () => {
+  const setup: Setup = new Setup();
```

Comment:
Jest suggests only initializing values inside `beforeEach` for safety / consistency reasons.